### PR TITLE
Fix debug panic

### PR DESCRIPTION
**Pull request by juanvallejo**
Debug was creating and using kcmd.LogOptions without passing a "consumeRequest" func and a RESTClientGetter

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1612661

cc @deads2k @soltysh

### CODE REVIEW

File: pkg/oc/cli/debug/debug.go [hunk 1]
```diff
@@ -828,3 +834,44 @@ func (o *DebugOptions) approximatePodTemplateForObject(object runtime.Object) (*
 
 	return nil, fmt.Errorf("unable to extract pod template from type %v", reflect.TypeOf(object))
 }
+
+// podContainerRunning returns false until the named container has ContainerStatus running (at least once),
+// and will return an error if the pod is deleted, runs to completion, or the container pod is not available.
+func podContainerRunning(containerName string) watch.ConditionFunc {
```
Comment (deads2k) [inline pkg/oc/cli/debug/debug.go]: are these the same?  oc/lib/conditions

Comment (juanvallejo) [reply pkg/oc/cli/debug/debug.go]: Thanks, moved this helper to the `oc/lib/conditions` package

File: pkg/oc/cli/admin/diagnostics/diagnostics/cluster/app_create/app.go [hunk 2]
```diff
@@ -124,3 +128,44 @@ There are many reasons why this can occur; for example:
 	`, now(), d.deployTimeout))
 	return false
 }
+
+// podContainerRunning returns false until the named container has ContainerStatus running (at least once),
+// and will return an error if the pod is deleted, runs to completion, or the container pod is not available.
+func podContainerRunning(containerName string) watch.ConditionFunc {
```
Comment (juanvallejo) [inline pkg/oc/cli/admin/diagnostics/diagnostics/cluster/app_create/app.go]: @deads2k this is a copy of the helper under oc/lib/conditions. This diagnostics file deals with internalversions, and we'll be removing diagnostics soon anyway, so I figured we could leave this copy here until then

**Pull request merged**